docker/data-science/notebooks/ch05/3.semantic-knowledge-graph.ipynb

```diff
@@ -332,47 +330,47 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 12,
+   "execution_count": 6,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
       "Simple Query Expansion:\n",
-      "{'query': \"vibranium vibranium^0.94237 wakandan^0.8197 adamantium^0.80724 wakanda^0.79122 alloy^0.75724 maclain^0.75623 klaw^0.75222 america's^0.74002 \", 'params': {'qf': 'title body', 'defType': 'edismax', 'mm': '0%'}}\n",
+      "{'query': \"vibranium vibranium^0.94237 wakandan^0.8197 adamantium^0.80724 wakanda^0.79122 alloy^0.75724 maclain^0.75623 klaw^0.75222 america's^0.74002 \", 'limit': 10, 'params': {'defType': 'edismax', 'ident': 'true', 'omitHeader': 'true', 'qf': ['title', 'body'], 'mm': '0%'}}\n",
       "\n",
       "Increased Precision, Reduced Recall Query:\n",
-      "{'query': \"vibranium vibranium^0.94237 wakandan^0.8197 adamantium^0.80724 wakanda^0.79122 alloy^0.75724 maclain^0.75623 klaw^0.75222 america's^0.74002 \", 'params': {'qf': 'title body', 'defType': 'edismax', 'mm': '30%'}}\n",
+      "{'query': \"vibranium vibranium^0.94237 wakandan^0.8197 adamantium^0.80724 wakanda^0.79122 alloy^0.75724 maclain^0.75623 klaw^0.75222 america's^0.74002 \", 'limit': 10, 'params': {'defType': 'edismax', 'ident': 'true', 'omitHeader': 'true', 'qf': ['title', 'body'], 'mm': '30%'}}\n",
```

Comment:
note: "indent" throughout as "ident"

Reply:
Created default schema for /select on applicable indexes

Reply:
It's still showing "ident" in github...